### PR TITLE
AG Grid Theme site docs

### PR DESCRIPTION
**Pull request by alycrys**

### CODE REVIEW

**Review by libertymayc — APPROVED** [review]

File: site/docs/components/ag-grid-theme/usage.mdx [hunk 1]
```diff
@@ -33,41 +33,77 @@ yarn add @salt-ds/ag-grid-theme
...
 
 To import the Salt AG Grid theme, use:
 
-```js
-import "@salt-ds/ag-grid-theme/salt-ag-theme.css";
+```
+import "@salt-ds/ag-grid-theme/salt.css";
 ```
 
 For light mode, use these CSS class names:
```
Comment (feliciackh) [inline site/docs/components/ag-grid-theme/usage.mdx:52]:
```suggestion
For light mode, use class name `.ag-theme-salt-light`
```

Comment (feliciackh) [reply site/docs/components/ag-grid-theme/usage.mdx:52]: nit

File: site/docs/components/ag-grid-theme/usage.mdx [hunk 2]
```diff
@@ -33,41 +33,77 @@ yarn add @salt-ds/ag-grid-theme
...
-.ag-theme-salt-high-compact-light
-.ag-theme-salt-high-light
-.ag-theme-salt-medium-light
-.ag-theme-salt-low-light
-.ag-theme-salt-touch-light
+```
+.ag-theme-salt-light
 ```
 
 For dark mode, use these CSS class names:
```
[the remark this answers is not in the document]
Comment (feliciackh) [reply site/docs/components/ag-grid-theme/usage.mdx:59]: nit

**Review by feliciackh — APPROVED** [review]

File: site/docs/components/ag-grid-theme/index.mdx [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 title: AG Grid Theme
 data:
-  description: AG Grid Theme is a package that adds Salt theme styles to AG Grid. The package contains CSS files only, and works with AG Grid versions 25 - 28.
+  description: AG Grid Theme is a package that adds Salt theme styles to AG Grid. The package contains CSS files only, and works with AG Grid versions 29 and 30.
```
Comment (feliciackh) [inline site/docs/components/ag-grid-theme/index.mdx:4]: qq - does this update no longer support versions 25-28?

**Pull request merged**